broker-daemon/models/block-order.js

```diff
@@ -172,6 +172,22 @@ class BlockOrder {
     })
   }
 
+  /**
+   * get boolean for if the blockOrder is a bid
```

Comment:
[nit] @martineehrlich we can omit docs for these `get` properties since they are self documenting